lib/fastlane/actions/verify_build.rb

```diff
@@ -67,7 +67,9 @@ def self.update_with_profile_info(app_path, values)
         values['provisioning_uuid'] = plist['UUID']
         values['team_name'] = plist['TeamName']
 
-        raise "Inconsistent identifier found" unless plist['Entitlements']['application-identifier'] == "#{values['team_identifier']}.#{values['bundle_identifier']}"
+        split_bundle_identifier = plist['Entitlements']['application-identifier'].partition('.').last
```

Comment:
It seemed like a convenient way to get the substring after the first occurrence of '.' For a bundle identifier like TEAMID.some.bundle.identifier, it returns some.bundle.identifier. I'm not a ruby expert, though, so if an implementation with split is desired I can certainly modify it.